docker/config/local/conseil/conseil.conf

```diff
@@ -24,10 +24,10 @@ platforms: {
 conseildb = {
   dataSourceClass = "org.postgresql.ds.PGSimpleDataSource"
   properties = {
-    databaseName="conseil-local"
+    databaseName="conseildatabase"
```

Comment:
why did this change? the idea is that "local" deploys a fully self-container env.

Reply:
need to find a way to use unpublished/specific credentials